pack/rpm.mk

```diff
@@ -27,6 +27,9 @@ PREBUILD_OS := prebuild-$(OS).sh
 PREBUILD_OS_DIST := prebuild-$(OS)-$(DIST).sh
 THEDATE := $(shell date +"%a %b %d %Y")
 
+ifdef CI
```

Comment:
Aren't it always set in `packpack` script?

https://github.com/packpack/packpack/blob/51d349a5953eca5f41db367a2940e2f83660d6f8/packpack#L22

Reply:
It is needed for manual run make from packpack, like this the issue exists for now:
```
rpmbuild \
	--define '_topdir /tarantool/build' \
	--define '_sourcedir /tarantool/build' \
	--define '_specdir /tarantool/build' \
	--define '_srcrpmdir /tarantool/build' \
	--define '_ci ' \
	--define '_builddir /tarantool/build/usr/src/debug' \
	-bs /tarantool/build/tarantool.spec
error: Macro %_ci has empty body
make: *** [/tarantool/packpack/pack//rpm.mk:112: /tarantool/build/tarantool-2.5.0.3-1.sle150000.src.rpm] Error 1
```

Reply:
Right, it sets in the packpack script, but rpmbuild fails anyway, and in general it is the problem of rpmbuild options. We can set double quotes right around the option variable, but we have there 2 common calls with it and I thought that it would be better to set quotes as variable value.

Reply:
> Aren't it always set in `packpack` script?
> 
> 
> 
> https://github.com/packpack/packpack/blob/51d349a5953eca5f41db367a2940e2f83660d6f8/packpack#L22


Right, it sets in the packpack script, but rpmbuild fails anyway, and in general it is the problem of rpmbuild options. We can set double quotes right around the option variable, but we have there 2 common calls with it and I thought that it would be better to set quotes as variable value.